test/protobuf/decoder_test.exs

```diff
@@ -126,7 +126,7 @@ defmodule Protobuf.Decoder.Test do
       use Protobuf, from: Path.expand("../proto/mumble.proto", __DIR__)
     end
 
-    msg = MumbleProto.Authenticate.new(username: "bitwalker")
+    msg = MumbleProto.MumbleProto.Authenticate.new(username: "bitwalker")
```

Comment:
There was never a time where:

```elixir
defmodule NotMyTopLevelModule do
  use Protobuf, "basic.proto", inject: true
end
```

would've resulted in `Basic.new`. It would've resulted in `NotMyTopLevelModule.new`.

It's also important to know that *all* Elixir modules have the `Elixir.*` prefix, as this is done by the Elixir compiler.